NineChronicles.Headless/GraphTypes/StateQuery.cs

```diff
@@ -55,6 +55,18 @@ public StateQuery()
                     return new WeeklyArenaState(
                         (Dictionary) context.Source.GetState(WeeklyArenaState.DeriveAddress(index)));
                 });
+            Field<AgentStateType>(
+                name: "agent",
+                arguments: new QueryArguments(new QueryArgument<AddressType>
```

Comment:
The `address` argument must be required.

```suggestion
                arguments: new QueryArguments(new QueryArgument<NonNullableGraphType<AddressType>>
```